tex/paper.tex

```diff
@@ -145,9 +145,8 @@ \section*{Introduction}
 cannot be reliably replicated, and therefore have not met the basic scientific
 criterion of falsifiability.
 
```

Comment:
Suggest instead:

To complicate matters further, a new strategy is often studied in isolation with opponents chosen by the creator of that strategy. As such, the results cannot be reliably replicated and therefore have not met the basic scientific criterion of falsifiability.